.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
move `.gitignore` to the root folder
and add exclusion for folders:

.idea/
target/

After that repository should look clean :)